src/jobs/release-package.yml

```diff
@@ -27,6 +27,10 @@ parameters:
     description: if true, the job will run but will not publish to npm or push to git
     default: false
     type: boolean
+  pre-job-steps:
+    description: steps that you want to run after the job is setup, but before installing dependencies (dryrun is ignored)
```

Comment:
I would simplify this to:

`steps that you want to run before installing dependencies (dryrun is ignored)`